src/main/java/com/google/devtools/build/lib/rules/android/AndroidConfiguration.java

```diff
@@ -861,9 +861,41 @@ public static class Options extends FragmentOptions {
           "--strategy=AndroidManifestMerger=worker",
           "--strategy=Aapt2Optimize=worker",
           "--strategy=AARGenerator=worker",
+          "--strategy=Desugar=worker",
+          "--strategy=DexBuilder=worker",
         })
     public Void persistentResourceProcessor;
 
+    @Option(
+        name = "persistent_multiplex_android_resource_processor",
```

Comment:
If this is meant to be used just while rolling the multiplex implementation, it ought to be an `experimental_` flag. If not, why not just set `support-multiplex-workers` in BusyBoxActionBuilder?

Reply:
I wouldn't be against just setting `support-multiplex-workers` everywhere but that might be on the riskier side given that 6.x will be the first release where there's actual multiplex support for all of the tools that the busybox wrapper runs.

Reply:
I don't think we want the flag around forever, so what is the testing strategy?

Reply:
Addressing a few things:

- There's a test for this in resource_processing_integration_test.sh, so I think that's covered
- I think setting `supports-multiplex-workers` in BusyBoxActionBuilder would also turn this on internally, and I don't think we want to sign up for figuring out how to roll that out right now. I think we should consider setting workers to be on for all this by default, but that can be a separate change
- I'm ok with leaving this without `experimental_`, because this isn't really a new thing that's being actively changed or developed, and it's not an API or similar

Reply:
I realized that with desugar and dexing in the list, this isn't strictly about resource processing. Maybe a more accurate name would be `--persistent_multiplex_android_tools`

Similarly, adding desugar and dexing to `--persistent_android_resource_processor` changes the behavior of the flag, and seems to make that name less accurate. We can't easily rename that though flag since it's been around for a while already. The solution would seem to be to introduce `--persistent_android_dex_desugar`.

If that sounds good I can make the change while importing this PR

Reply:
Yeah so even though we were trying to simplify things, I seem to be now proposing:
```
--persistent_android_resource_processor
--persistent_multiplex_android_resource_processor
--persistent_android_dex_desugar
--persistent_multiplex_android_dex_desugar
--persistent_multiplex_android_tools
```
`--persistent_android_resource_processor` and `--persistent_multiplex_android_tools` adds all the previous ones (directly or indirectly).

But this seems to be the most accurate and safest.